schema/iguana.owl

```diff
@@ -1,4 +1,3 @@
-
 <!DOCTYPE rdf:RDF [
     <!ENTITY dcterms "http://purl.org/dc/terms/" >
     <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
```

Comment:
Please add the following imports:
```
        <owl:imports rdf:resource="http://purl.org/dc/elements/1.1/"/>
        <owl:imports rdf:resource="http://purl.org/dc/terms/"/>
        <owl:imports rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#"/>
        <owl:imports rdf:resource="http://www.w3.org/2000/01/rdf-schema#"/>
        <owl:imports rdf:resource="http://www.w3.org/2002/07/owl"/>
```

in `<owl:Ontology ...>`

Also, please double check if I missed any. Ontologies do typically not import any vocabulary.